src/rockstor/storageadmin/static/storageadmin/js/views/share_details_layout_view.js

```diff
@@ -94,6 +94,10 @@ ShareDetailsLayoutView = RockstorLayoutView.extend({
 		var current_appliance = this.appliances.find(function(appliance) {
 			return appliance.get('current_appliance') == true;
 		});
+        var systemShare = false;
+        if (parseInt(this.share.get('id')) < 3 && this.share.get('pool')['id'] == '1') {
```

Comment:
956c66b got it on both, you're totally right - my fault, I really didn't remember about that role field
M.